src/net/sourceforge/kolmafia/request/MallSearchRequest.java

```diff
@@ -254,7 +233,10 @@ public void run() {
     // If an exact match, we can think about updating mall_price().
     if (this.searchString.startsWith("\"") && this.results.size() > 0) {
       AdventureResult item = this.results.get(0).getItem();
-      MallPriceManager.maybeUpdateMallPrice(item, new ArrayList<PurchaseRequest>(this.results));
+      // This was "maybe" updateMallPrice - i.e., only do so if there is not
```

Comment:
I feel like this comment is only valuable in the context of this PR -- seeing what the code was, and what it changed into.

I don't feel like it adds any value to the codebase after this PR goes in. Can it be removed?

Reply:
Done.